spec/pubid_iso/identifier_spec.rb

```diff
@@ -249,4 +249,13 @@
     it_behaves_like "converts pubid to urn"
     it_behaves_like "converts pubid to pubid"
   end
+
+  context "ISO/IEC PDTR 20943-5" do
+    let(:original) { "ISO/IEC PDTR 20943-5" }
+    let(:pubid) { "ISO/IEC TR 20943-5" }
```

Comment:
"PDTR" and "PDTS" means "CD" stage. This should be "ISO/IEC CD TR 20943-5"

Need to also add a test for PDTS.